src/solver/utils/ortools_utils.cpp

```diff
@@ -98,6 +103,8 @@ void ProblemSimplexeNommeConverter::TuneSolverSpecificOptions(MPSolver* solver)
     default:
         break;
     }
+    // How / When to check if user has input wrong parameters ?
+    solver->SetSolverSpecificParametersAsString(solverParameters);
```

Comment:
This function returns `true` in case of success, `false` in case of error. Also, make sure to pass `THREADS 1`, otherwise XPRESS may to run multiple threads by default. https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/THREADS.html